Source/Diagnostics/ReducedDiags/BeamRelevant.cpp

```diff
@@ -198,14 +199,14 @@ void BeamRelevant::ComputeDiags (int step)
 
         using PType = typename WarpXParticleContainer::SuperParticleType;
 
-        amrex::ReduceOps<ReduceOpSum,ReduceOpSum,ReduceOpSum,ReduceOpSum,ReduceOpSum,
-        ReduceOpSum,ReduceOpSum,ReduceOpSum> reduce_ops;
-        auto r = amrex::ParticleReduce<amrex::ReduceData<ParticleReal,ParticleReal,
-        ParticleReal,ParticleReal,ParticleReal,ParticleReal,ParticleReal,ParticleReal>>(
+        // number of reduction operations in first concurrent batch
+        const int num_red_ops_1 = 8;
```

Comment:
```suggestion
        constexpr std::size_t num_red_ops_1 = 8;
```
Looks great! I would write constexpr explicitly here (and in the other places including in ImpactX). Currently, constexpr is implied by the const; for the code to function constexpr is necessary. For MSVC I believe size_t is needed.

Reply:
Thanks, @AlexanderSinn!